doc/nrf/ug_matter_overview_dfu.rst

```diff
@@ -0,0 +1,189 @@
+.. _ug_matter_overview_dfu:
+
+Matter OTA
+##########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter Over-The-Air is a process that allows a Matter device in a Matter fabric to update its firmware.
+Most of the Matter devices will require such an update at some point during their lifecycle.
+The software image is provided by another Matter node, which acquired information about it and downloaded it from a centralized and reliable source.
+The download takes place over a special transfer protocol and user consent is required for the image to be applied on the target Matter device.
+
+.. _ug_matter_overview_dfu_roles:
+
+Matter OTA roles
+****************
+
+The following roles participate in the Matter OTA process:
+
+OTA Requestor
+   Any Matter device that is going to have its firmware updated.
+   This role corresponds to the OTA Requestor :ref:`device type <ug_matter_overview_data_model>`, which implements the client role for the OTA Software Update Provider cluster and the server role for the OTA Software Update Requestor cluster.
+
+   The OTA Requestor can only receive information about available software updates from the OTA Provider.
+   Because of this, it must implement the mandatory mechanism to discover providers and query the OTA Provider periodically about new software images using the mandatory polling mechanism.
+   However, it can also receive information about the OTA Providers from the commissioner during :ref:`ug_matter_overview_commissioning`.
+
+OTA Provider
```

Comment:
Please add information that OTA requestor is typically implemented on smart home hubs from Ecosystem providers.